dleq/dleq.go

```diff
@@ -1,3 +1,4 @@
+// Package dleq provides the sub-api that we need from the go-dleq package.
```

Comment:
"sub-api" is too vague. It should be describing exactly what the functionality provided to the atomic swap protocol is from a high level. I'll work with you on this unless you have a suggestion off the top of your head.